packages/mobile/android/app/src/main/java/com/quietmobile/MainActivity.java

```diff
@@ -104,10 +104,13 @@ protected void onCreate(Bundle savedInstanceState) {
         Intent intent = getIntent();
         checkAgainstIntentUpdate(intent);
 
-        if (BuildConfig.SHOULD_RUN_BACKEND_WORKER == "true") {
-            Context context = getApplicationContext();
-            new BackendWorkManager(context).enqueueRequests();
-        }
+        Context context = getApplicationContext();
+        new BackendWorkManager(context).enqueueRequests();
+
+//        if (BuildConfig.SHOULD_RUN_BACKEND_WORKER == "true") {
```

Comment:
The problem with this conditional statement is that detox has problems reading the environment variable. I'd want to see how it works with storybook before removing it entirely.